calcom/cal.com
# fix: Show messaging app in locations list

## What does this PR do?

Fix taken from https://github.com/calcom/cal.com/pull/10653 to quickly fix the bug mentioned here.

![image](https://github.com/calcom/cal.com/assets/1780212/47703887-2081-4d13-bc27-6de121f8900e)

Fixes # (issue)

## Type of change




## How should this be tested?
- Install a messaging app and verify that you can add that app in an EventType and book the event with that location

## Mandatory Tasks


## Checklist
- I haven't added tests that prove my fix is effective or that my feature works
